desc/examples/precise_QA.py

```diff
@@ -0,0 +1,88 @@
+"""Example script for recreating the precise QA configuration of Landreman and Paul."""
+
+from desc import set_device
+
+set_device("gpu")
+
+import numpy as np
+
+from desc.continuation import solve_continuation_automatic
+from desc.equilibrium import EquilibriaFamily, Equilibrium
+from desc.geometry import FourierRZToroidalSurface
+from desc.grid import LinearGrid
+from desc.objectives import (
+    AspectRatio,
+    FixBoundaryR,
+    FixBoundaryZ,
+    FixCurrent,
+    FixPressure,
+    FixPsi,
+    ForceBalance,
+    ObjectiveFunction,
+    QuasisymmetryTwoTerm,
+    RotationalTransform,
+)
+from desc.optimize import Optimizer
+
+# create initial equilibrium
+surf = FourierRZToroidalSurface(
+    R_lmn=[1, 0.166, 0.1],
+    Z_lmn=[-0.166, -0.1],
+    modes_R=[[0, 0], [1, 0], [0, 1]],
+    modes_Z=[[-1, 0], [0, -1]],

```

Comment:
We should be able to get this precise QA result starting from an axisymmetric initial equilibrium.

Reply:
I had tried that and it occasionally got stuck as a tokamak depending on how the objectives were weighted, starting from non-axisymmetry seemed to converge to something good more robustly regardless of weighting.

Reply:
Is there a set of weights that does make it converge? I thought it worked in the past. It would be nice to show we can reproduce the result starting from the same initial condition as SIMSOPT.

Reply:
I'm having trouble finding a case that starts from axisymmetry and converges correctly. I remember we had similar issues in trying to perturb a tokamak into a stellarator to reduce current, it seems there's some deep local minimum at axisymmetry. 

Another possibility is that its an issue with vacuum tokamaks not really being well defined, so the initial equilibrium is weird in some way.